flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/RuntimeMonitorHandlerBase.java

```diff
@@ -67,6 +67,8 @@ public RuntimeMonitorHandlerBase(
 		this.httpsEnabled = httpsEnabled;
 	}
 
+	public abstract String[] getPaths();
```

Comment:
I know the other methods are also not commented well, but could you add comments here?